azure-devops/azext_devops/dev/repos/import_request.py

```diff
@@ -33,7 +33,8 @@ def create_import_request(git_source_url, project=None, repository=None,
             detect=detect,
             organization=organization,
             project=project,
-            repo=repository)
+            repo=repository,
+            repo_required=True)
```

Comment:
Should this be true for some other repo commands as well?

Reply:
maybe, but I am not very sure right now